proposals/UX.md

```diff
@@ -0,0 +1,45 @@
+# Application form for starting a SIG
+
+## What will be the name of the SIG
+
+User Experience SIG:  **UX-SIG**
+![Creativity](https://positivepsychology.com/wp-content/uploads/boost-your-creativity.jpg)
+
+## Which two persons will act as SIG Lead?
+
+- Jesus Garcia Gonzalez
+- Sven van der Burg
+
+
+## What is the SIG's mission?
```

Comment:
I think we can be more bold here. Something like this: Our mission is to make development at NLeSC more user-centric, thereby significantly improving the value of project outcomes.

Reply:
I loved the bold statement. I put it in the document